WordPress/src/main/java/org/wordpress/android/ui/mysite/items/SiteItemsBuilder.kt

```diff
@@ -38,7 +38,6 @@ class SiteItemsBuilder @Inject constructor(
                 params.activeTask == QuickStartTask.REVIEW_PAGES
 
         return listOfNotNull(
-                siteListItemBuilder.buildPlanItemIfAvailable(params.site, showExplorePlansFocusPoint, params.onClick),
```

Comment:
np: We can also remove the line 35, since it's redundant now.

Reply:
Thanks for spotting! Removed now in 9d60b814778680e10742c4e402c4902faba6bc3a